### ISSUE TITLE
tests.test_cookies.TestCookies.test_request_cookie_parsing fails on Python 3.5

### ISSUE THREAD

**Issue by carlwgeorge** [falconry/falcon#654]
This test fails on Python 3.5.  The error is reproducible on Fedora Rawhide and Arch Linux.

```
tests.test_cookies.TestCookies.test_request_cookie_parsing ... ERROR
```
```
======================================================================
ERROR: tests.test_cookies.TestCookies.test_request_cookie_parsing
----------------------------------------------------------------------
testtools.testresult.real._StringException: Traceback (most recent call last):
  File "/builddir/build/BUILD/falcon-0.3.0/tests/test_cookies.py", line 140, in test_request_cookie_parsing
    self.assertEqual("no", req.cookies["logged_in"])
KeyError: 'logged_in'
```

**Comment by rikuayanokozy** [comment]
```
tests.test_cookies.TestCookies.test_request_cookie_parsing ... ok
```

The test is work fine with latest code.

**Comment by kgriffs** [comment]
I believe this is fixed now in master and will go out in the 0.4 release. Please reopen if you find this is not the case. Thanks!
